modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+[id="updating-eus-to-eus-upgrade_{context}"]
+= EUS to EUS upgrade
+
+The following procedure pauses all non-master MachineConfigPools and performs upgrades from 4.8 to 4.9 to 4.10, then unpauses the previously paused MachineConfigPools.
+Following this procedure reduces the total upgrade duration and the number of times worker nodes are restarted.
+
+.Prerequisites
+
+* Review the release notes for {product-title} 4.9 and 4.10
+* Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to upgrading from {product-title} 4.8 to 4.9.
+* Verify that your cluster is not running {product-title} 4.8.14 version or earlier.
+If your cluster is running {product-title} 4.8.14 or earlier, you must upgrade to a later 4.8.z version prior to upgrading to 4.9.
+The upgrade to 4.8.14 or later which is necessary to fulfill the minimum version requirements that must be performed without pausing MachineConfigPools.
+* Verify that MachineConfigPools is unpaused.
+
+.Procedure
+
+. Verify that all MachineConfigPools display a status of `UPDATED` and no MachineConfigPools display a status of `UPDATING`.
+To view the status of all MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. To pause the MachineConfigPools you wish to skip reboots on, run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+$ oc get mcp -o
+----
++
+.Example output
++
+[source,terminal]
+----
+custom-columns=NAME:.metadata.name,PAUSED:.spec.paused
+NAME 	    PAUSED
+master      false
+worker      true
+----
+
+. To change to the `eus-4.10` channel and upgrade to 4.9, run the following commands:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.10
+$ oc adm upgrade --to-latest
```

Comment:
When it's a disconnected cluster, we can not do upgrade with --to-latest directly. Maybe we could give a link here for users to redirect to upgrade in disconnected cluster section.

Reply:
make sense. ack.